### DISCUSSION TITLE
Does file globbing work in "format-resources:" in `_extension.yml`?

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#6229] [Q&A]
### Description

I am bit confused about where file globbing works. 

In Quarto, it is not the case that it should work in all YAML files, is it? 

I might be doing something wrong but it seems that in `format-resources` in `_extension.yml`, it doesn't.

**Comment by dragonstyle** [comment]
It looks like `format-resources` doesn't currently support globs, though I agree that this is confusing and it should. I'll open an issue and try to get that fixed soon.

**Comment by mcanouil** [comment]
Closing this thread in favour of #6241.
